asr1k_neutron_l3/models/asr1k_pair.py

```diff
@@ -27,13 +27,7 @@
 
 
 class ASR1KContextBase(object):
```

Comment:
Yeah, we could just remove that class. I left it there, as I "just removed the code we don't need" and thought a common base class wouldn't be to bad either way, maybe we want to refactor some of the common functionality into that one.

I can remove it if you'd like, otherwise I'd just leave it for now.